docs/content/latest/benchmark/tpcc-ysql/_index.md

```diff
@@ -57,7 +66,7 @@ You will need the IP addresses of the nodes in the cluster for the next step.
 {{< /tip>}}
 
 
-### Configure DB connection params (optional)
+## 2. Configure DB connection params (optional)
 
 If not working with the defaults, we can change the username, password, port, etc. using the configuration file at `config/workload_all.xml`. We can also change the terminals or the physical connections being used by the benchmark using the configuration.
 
```

Comment:
Let's remove the options that don't need to be changed by users - specifically, `dbtype`, `driver`, `terminals` and `numDBConnections`.

terminals and numDBConnections is something that we should set automatically based on number of warehouses. It's suboptimal and unnecessary for users to change this.

Please also add command line options like loader threads and warehouses to this list and change the title to `Configure workload parameters`